docs/ConfigurationOptions/index.md

```diff
@@ -29,6 +29,7 @@ grate --connectionstring="Server=(localdb)\MSSQLLocalDB;Integrated Security=true
 | -ct<br>--commandtimeout &lt;commandtimeout&gt; | 60s | This is the timeout when commands are run. This is not for admin commands or restore. |
 | -cta<br>--admincommandtimeout &lt;admincommandtimeout&gt; | 300 | This is the timeout when administration commands are run (except for restore, which has its own) |
 | --databasetype<br>--dbt<br>--dt <mariadb \| oracle \| postgresql \| sqlite \| sqlserver> | sqlserver | Tells grate what type of database it is running on. |
+| --createDatabaseCustomScript<br>--cds | null | Tells grate to use this script for creating a database instead of the default based on the DatabaseType. |
```

Comment:
All the other options are lower case only.  Could we do the same here for consistency please?